conf/application.conf

```diff
@@ -101,14 +101,33 @@ google-analytics {
   host = auto
 }
 
+
+# Suppress the auditing of assets as per PLATUI-837
+controllers {
+  controllers.Assets = {
+    needsAuditing = false
+  }
+  uk.gov.hmrc.govukfrontend.controllers.Assets = {
+    needsAuditing = false
+  }
+  uk.gov.hmrc.hmrcfrontend.controllers.Assets = {
+    needsAuditing = false
+  }
+}
+
 assets {
   version = "3.11.0"
   version = ${?ASSETS_FRONTEND_VERSION}
   url = "http://localhost:9032/assets/"
 }
 
+play.assets.cache."/public/"="no-cache, max-age=0"
```

Comment:
These cache directives shouldn't be needed, tracking.js is not hosted by this microservice.